### PR TITLE
add new cite.html

### PR DESCRIPTION
**Pull request by tjhei**

### CODE REVIEW

**Comment by tjhei** [comment]
As discussed during the meeting today. I will follow up with PRs for linking to it on the main website, readme/manual/etc.

File: cite.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+Computations were done using the ASPECT code version 1.5.0, see \cite{heister_aspect_methods2,...}.</code></pre>
+	    </p>
+
+	    <p><b>2. Acknowledgement</b>
+	      <pre><code>
+We thank the Computational Infrastructure for Geodynamics (geodynamics.org) which is funded by the National Science Foundation under award EAR-0949446 and EAR-1550901 for supporting the development of ASPECT.</code></pre>
+	    </p>
+
+	    <p><b>3. Citation</b><br>
+	      Please cite one of all of the following documents. The canonical reference for this software:
```
Comment (jdannberg) [inline cite.html]: one _or_ all

Comment (jdannberg) [reply cite.html]: And can you put the sentence "The canonical reference for this software" into a new line?

File: index.html [hunk 2]
```diff
@@ -111,6 +111,18 @@ <h2>
 		</td>
 	      </tr>
 
+	      <tr>
+		<td style="vertical-align:top;height:4em">
+		  <a class="btn btn-success" href="cite.html">Cite! &raquo;</a>
```
Comment (jdannberg) [inline index.html]: The other green boxes all have a different size. I think that's because they have the  `style="width:100%" ` code? Can you fix that?

**Comment by tjhei** [comment]
fixed. I am merging so we can take a look at it...

**Pull request merged**